fastlane/Fastfile

```diff
@@ -41,229 +56,101 @@ platform :ios do
   end
 
   lane :qa_release do
-    time = Time.new
-    str_time = time.strftime("%Y-%m-%d %H:%M:%S")
-    # Set timestamp as version_code to differentiate builds in firebase
-    build_number = time.to_i
-
-    appetize_api_token = ENV['APPETIZE_API_TOKEN']
-    sdk_version_name = ENV['SOURCE_BRANCH']
-
-    # cocoapods(
-    #   clean_install: true,
-    #   podfile: "./Example/Podfile"
-    # )
-
-    # Set version number
-    increment_version_number(
-      version_number: sdk_version_name,
-      xcodeproj: "./Example/PrimerSDK.xcodeproj"
-    )
-
-    increment_build_number(
-      build_number: build_number,
-      xcodeproj: "./Example/PrimerSDK.xcodeproj"
-    )
 
-    create_keychain(
-      name: ENV["MATCH_KEYCHAIN_NAME"],
-      password: ENV["MATCH_KEYCHAIN_PASSWORD"],
-      default_keychain: true,
-      unlock: true,
-      timeout: 3600,
-      lock_when_sleeps: true
-    )
+    set_version_and_build_number
 
-    match(
-      type: "development",
-      readonly: true,
-      keychain_name: ENV["MATCH_KEYCHAIN_NAME"],
-      keychain_password: ENV["MATCH_KEYCHAIN_PASSWORD"],
+    setup_signing(
+      match_type: "development"
     )
 
-    # Build for appetize
```

Comment:
Is Appetize removed?

Reply:
No, Appetize is still there and it actually works 👍 
It is now part of a `private_lane`. Same for Browserstack.